keyboards/mechlovin/hannah65/rev1/readme.md

```diff
@@ -0,0 +1,15 @@
+# hannah65 rev1
+
+![hannah65](https://i.imgur.com/qf0BYI2l.png)
+
+A 65% PCB, compatible with E6.5, Kyuu,...
+
+* Keyboard Maintainer: [Team Mechlovin](https://github.com/mechlovin)
+* Hardware Supported: Hannah65, STM32F303
+* Hardware Availability: [Mechlovin Store](https://mechlove.com/)
+
+Make example for this keyboard (after setting up your build environment):
+
+    make mechlovin/hannah65/rev1:default
+
```

Comment:
```suggestion

Flashing example for this keyboard:

    make mechlovin/hannah65/rev1:flash
    
```

Reply:
Can you also add a line on how to put the board into bootloader mode? (if there is a switch, how to use it, etc)